### PR TITLE
fix pct tests failing

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: olivier lamy <olamy@apache.org>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -400,10 +394,25 @@
...
+      <version>1.20</version>
+      <scope>test</scope>
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>jenkins-test-harness</artifactId>
```
Comment (warden) [inline pom.xml]: Do we need this if this is part of https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L59 ?

Comment (olamy) [reply pom.xml]: this is fixing enforcer upperBoundDependency

**Comment by warden** [comment]
I'd like to do a release tomorrow (and try using GitHub actions for this) - are you ok with me doing the release or do you want to review any more pending stuff?

**Comment by olamy** [comment]
I would prefer doing if you don't mind. Using something gh actions means sharing jenkins deployment credentials etc... For security reasons I'm not sure if it's really safe...

**Comment by warden** [comment]
I don't mind, go ahead :) About the credentials, let's continue the discussion in projects.

**Pull request merged**